courses/machine_learning/tensorflow/a_tfstart.ipynb

```diff
@@ -4,29 +4,39 @@
    "cell_type": "markdown",
```

Comment:
<p>Please change Tensorflow to TensorFlow in the title.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1015/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>